File: Leetcode/Python/3110. Score of a String/solution.py
# https://leetcode.com/problems/score-of-a-string/description

class Solution:
    def scoreOfString(self, s: str) -> int:
        score = 0

        for idx in range(1, len(s)):
            score += abs(ord(s[idx]) - ord(s[idx - 1]))

        return score

File: Leetcode/Typescript/3110. Score of a String/solution.ts
// https://leetcode.com/problems/score-of-a-string/description

function scoreOfString(s: string): number {
  let score = 0;

  for (let idx = 1; idx < s.length; idx++) {
    score += Math.abs(s[idx - 1].charCodeAt(0) - s[idx].charCodeAt(0));
  }
  return score;
}
